content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -23,11 +23,13 @@ further_reading:
 Front-end errors are collected with Real User Monitoring (RUM). The error message and stack trace are included when available.
 
 ## Error origins
-Front-end errors are split into four different categories depending on their `origin`:
+Front-end errors come from several different origins:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since we have renamed error origin into error source in 2020, it could be nice to reflect it as well 🙂